src/java.base/share/native/libjli/java.c

```diff
@@ -534,6 +545,260 @@ invokeInstanceMainWithoutArgs(JNIEnv *env, jclass mainClass) {
     return 1;
 }
 
+#if defined(J9VM_OPT_CRAC_SUPPORT)
+static char *getCommandLineOptionValue(const char *optionName, int argc, char **argv, int *error) {
+    *error = 0;
+    for (int i = 0; i < argc; i++) {
+        char *arg = argv[i];
+        if (0 == strncmp(arg, optionName, strlen(optionName))) {
```

Comment:
This might return an incorrect result when the arg is something like `-XX:CRaCRestoreFromExtra`.